CLI 1.3.0, 0.29.1, 0.28.2 release notes (#906)
astronomer/docs: software/custom-image-registry.md

- Helm.
- kubectl.

> - Astro CLI version 1.0.3+.

Reviewer: ```suggestion
- Astro CLI version 1.3.0+.
```

## Push code to a custom registry

> You can use the Astro CLI to push build and push images from a Kubernetes cluster to your custom registry. Astronomer Software automatically recognizes that you pushed the image and pulls the image from the custom registry onto your Deployment.

Author: ```suggestion
You can use the Astro CLI to push build and push images from a Kubernetes cluster to your custom registry. Based on the Helm configurations in your Kubernetes cluster, the Astro CLI automatically detects your custom image registry and pushes your image to it. It then calls the Houston API to update your Deployment to pull the new image from the registry
```

> You can use the Astro CLI to push build and push images from a Kubernetes cluster to your custom registry. Based on the Helm configurations in your Kubernetes cluster, the Astro CLI automatically detects your custom image registry and pushes your image to it. It then calls the Houston API to update your Deployment to pull the new image from the registry

Author: ```suggestion
You can use the Astro CLI to push build and push images from a Kubernetes cluster to your custom registry. Based on the Helm configurations in your Kubernetes cluster, the Astro CLI automatically detects your custom image registry and pushes your image to it. It then calls the Houston API to update your Deployment to pull the new image from the registry.
```
